aviatrix/resource_aviatrix_transit_external_device_conn.go

```diff
@@ -674,11 +674,16 @@ func resourceAviatrixTransitExternalDeviceConnRead(d *schema.ResourceData, meta
 		id := d.Id()
 		log.Printf("[DEBUG] Looks like an import, no 'connection_name' or 'vpc_id' received. Import Id is %s", id)
 		parts := strings.Split(id, "~")
```

Comment:
Maybe we can use [strings.SplitN ](https://pkg.go.dev/strings#SplitN) here so that the rest of the code does not need to be changed.
`parts := strings.SplitN(id, "~", 2)`